vars/pmmVersion.groovy

```diff
@@ -45,11 +45,12 @@ def call(String type='latest') {
     '2.33.0': 'ami-07e19e90c30c6989c',
     '2.34.0': 'ami-06dd2cfc9b70bb79b',
     '2.35.0': 'ami-003bff3e1c59f54b3',
+    '2.36.0': 'ami-0ce04c507ec1187b1',
   ]
 
   List<String> versionsList = new ArrayList<>(versions.keySet());
   // Grab 10 latest versions
-  List<String> ovfVersions = ['2.35.0', '2.34.0', '2.33.0', '2.32.0', '2.31.0', '2.30.0', '2.29.1', '2.28.0', '2.27.0', '2.26.0', '2.25.0', '2.24.0'];
+  List<String> ovfVersions = ['2.36.0', '2.35.0', '2.34.0', '2.33.0', '2.32.0', '2.31.0', '2.30.0', '2.29.1', '2.28.0', '2.27.0', '2.26.0', '2.25.0', '2.24.0'];
```

Comment:
@vyarosh remove one older version, add one new to OVF list

Reply:
done